run_testlist.py

```diff
@@ -952,19 +972,27 @@ def _expand_variant(self, env, ppc, variant):
         self._add_partition(env, nodes)
         self._add_timeout(env, timeout)
 
+    def _ec_cell_size_variant(self, env, variant):
+        env['EC_CELL_SIZE'] = str(variant)
+
     def run(self):
         for test in self._testlist:
             if not test['enabled']:
                 continue
 
+            #Set default EC cell size variants
+            if 'ec_cellsize_variants' not in test:
+                test['ec_cellsize_variants'] = ['default']
+
             env = self._env
             self._expand_default_env_vars(env, test)
             self._expand_extra_env_vars(env, test)
             ppc = test['ppc']
             for variant in test['testvariants']:
                 self._expand_variant(env, ppc, variant)
-                subprocess.Popen(self._script, env=env)
-
+                for ec_variant in test['ec_cellsize_variants']:
+                    self._ec_cell_size_variant(env, ec_variant)
+                    subprocess.Popen(self._script, env=env)
```

Comment:
Making sure I understand this correctly... this runs each testvariant with each ec_cellsize_variant?

Reply:
Correct.